tests/test_add_takeoff_and_landing_command.py

```diff
@@ -0,0 +1,100 @@
+"""
+Test process
+"""
+
+import dronekit
+import pytest
+
+from modules import add_takeoff_and_landing_command
+
+
+ALTITUDE = 50
+MAVLINK_TEST_COMMAND = dronekit.mavutil.mavlink.MAV_CMD_NAV_WAYPOINT
+MAVLINK_TEST_FRAME = dronekit.mavutil.mavlink.MAV_FRAME_GLOBAL
+
+
+@pytest.fixture
+def non_empty_commands():
+    """
+    Fixture for a list of commands.
+    """
+    commands = [
+        dronekit.Command(
+            0,
+            0,
+            0,
+            MAVLINK_TEST_FRAME,
+            MAVLINK_TEST_COMMAND,
+            0,
+            0,
+            0,  # param1
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+        ),
+        dronekit.Command(
+            0,
+            0,
+            0,
+            MAVLINK_TEST_FRAME,
+            MAVLINK_TEST_FRAME,
+            0,
+            0,
+            0,  # param1
+            0,
+            0,
+            0,
+            0,
+            0,
+            0,
+        )
+    ]
+    return commands
+
+
+@pytest.fixture
+def empty_commands():
+    """
+    Fixture for an empty list of commands.
+    """
+    commands = []
+    return commands
+
+
+def assert_takeoff_and_landing_commands(commands_input, commands_actual, altitude):
```

Comment:
Comparison order is actual then expected (e.g. `assert actual == expected` ), change signature to follow this order.

Reply:
Actually, remove `commands_input` if it's not being used.